### ISSUE TITLE
server crashes giving unexpected token in json error

### ISSUE THREAD

**Issue by NikosEfthias** [elm-tooling/elm-language-server#120]
![image](https://user-images.githubusercontent.com/9097437/62053409-320e3400-b220-11e9-8269-73135643300b.png)

![image](https://user-images.githubusercontent.com/9097437/62054094-95e52c80-b221-11e9-897f-7592c5dc39cb.png)

## Expected Behavior
it should just work

## Current Behavior
server crashes 
## Possible Solution
Tried both in linux and mac and also tried installing through npm and building from source
nothing chages the result is always the same
## Steps to Reproduce (for bugs)
1. install the elm lsp as suggested on Readme using npm
2. add following to coc-settings.json
```json
	"elmLS": {
			"command": "elm-language-server",
			"args": ["--stdio"],
			"filetypes": ["elm"],
			"rootPatterns": ["elm.json"],
			"initializationOptions": {
				"runtime": "node",
				"elmPath": "elm",
				"elmFormatPath": "elm-format",
				"elmTestPath": "elm-test"
			}
		}
```
3. start an elm project
4. run `:CocList symbols` (or anything else actually")

## Context
I was just trying to open symbols table
## Your Environment
* Version used: Latest master version
* Editor used: neovim
* Environment name and version (e.g. node.js 5.4): `node 10.16, elm 0.19, elm-format 0.8.1`
* Operating System and version:
    `Arch Linux with zen kernel`

**Comment by Razzeee** [comment]
Can you try with the elm 0.19.1 alpha-4 releases? There were some json escape errors that got fixed in the api.

**Comment by andys8** [comment]
Maybe the language server could have better error messages when JSON can't be parsed. And also not crash.

**Comment by NikosEfthias** [comment]
@Razzeee I am not sure how i can get different version of elm. I have checked their github repo but i dont know what to look for. Should i download the compiler and compile it by myself?

**Comment by Razzeee** [comment]
@NikosEfthias if you can, please check the #core-coordination channel in the elm slack for the binaries, as it's only for testing right now, it's tried to not spread them too far.

**Comment by Razzeee** [comment]
Closing due to missing feedback.

**Comment by jschomay** [comment]
Just cross posting that this issue has been fixed in d6fe77c.  See #161.
